src/vstart.sh

```diff
@@ -1596,7 +1596,7 @@ do_rgw_create_users()
         --access-key ABCDEFGHIJKLMNOPQRST \
         --secret abcdefghijklmnopqrstuvwxyzabcdefghijklmn \
         --display-name youruseridhere \
-        --email s3@example.com -c $conf_fn > /dev/null
+        --email s3@example.com --caps="user-policy=*" -c $conf_fn > /dev/null
```

Comment:
Before when I was testing, I could see failures in some s3-tests due to addition of this parameter. So, can you please double check it before merging?

Reply:
sure, thanks for pointing this out.

Reply:
I didnt see any extra failures with this branch vs main.